trainAdamWC1.py

```diff
@@ -249,9 +249,11 @@ def train_eval(model,val_data,batch_size_val,epoch_no):
             loss_l, loss_c, loss_landm = criterion(out, priors, targets_)
             loss = cfg['loc_weight'] * loss_l + loss_c + loss_landm
             loss_val += loss.item()
-
-    # loss_val = loss_val / (batch_size * len(val_data))  # get average loss per image
-    print('Validation loss for Epoch {} : {}'.format(epoch_no,loss_val))
+    if not is_ohem:
+        # loss_val = loss_val / (batch_size * len(val_data))  # get average loss per image
+        print('Validation loss for Epoch {} : {}'.format(epoch_no,loss_val))
+    else:
+        print('Ohem loss for Epoch {} : {}'.format(epoch_no,loss_val))
```

Comment:
Did you add the HNM images in the data folder with older training images??

Reply:
No not yet.

Reply:
I mean you just need add those new images in `train` folder as well as add those images to a new folder named `ohem`. That's it. It'll work

Reply:
So right now the train loss does not contain loss coming from `hard negative examples`?

Reply:
`train_loss` is from both `train` and `ohem` images , as it should be. But `ohem_loss` is from `ohem` images only because we want to track those.